conf/nutch-default.xml

```diff
@@ -1390,7 +1390,7 @@ For more detailed information on the working of this filter
 visit https://wiki.apache.org/nutch/SimilarityScoringFilter-->
 
 <property>
-    <name>scoring.similarity.model.path</name>
+    <name>cosine.goldstandard.file</name>
     <value>goldstandard.txt</value>
     <description>Path to the gold standard file which contains all the relevant text and terms, 
```

Comment:
It is a goldstandard file now and the model containing the term vector gets created at runtime. I felt the earlier name was misleading and given that we could have more similarity metrics in the future,  I renamed it. 
Later on if we can incorporate the idea of a corpus of documents, we can rename it to something like cosine.corpus.path ?